src/docs/src/replication/replicator.rst

```diff
@@ -252,16 +252,16 @@ normal.
 Documents describing the same replication
 =========================================
 
-Lets suppose 2 documents are added to the ``_replicator`` database in
+Let's suppose 2 documents are added to the ``_replicator`` database in
 the following order:
 
 .. code-block:: javascript
 
     {
         "_id": "my_rep",
-        "source": "http://myserver.com/foo",
-        "target":  "http://user:pass@localhost:5984/bar",
-        "create_target":  true,
+        "source": "http://adm:pass@myserver.com/foo",
```

Comment:
In a few cases, where we have a remote endpoint like "myserver.com" we may want to use a different password user combination to illustrate that those endpoint may have a completely different set of credentials. So in this case we could have source have a `user:password` for instance

Reply:
Yeah, that were my thoughts too, but great work @jiahuili430 so far, it's almost ready already...

Reply:
Changed, thanks for the review!